### ISSUE TITLE
MNIST Training Remains Random

### ISSUE THREAD

**Issue by JetJadeja** [JetJadeja/brainwave#1]
When training the model highlighted in the example code (in the README.MD), which I have copied below, the accuracy does not seem to move far beyond 10%, which suggests that the model is not learning. However, the weights do seem to be changing, so I am not entirely sure why the accuracy does not seem to be changing.

```python
from brainwave.model import Sequential
from brainwave.layers import Dense, Flatten, Dropout
from brainwave.datasets import mnist
import numpy as np

# Load the MNIST dataset.
(x_train, y_train), (x_test, y_test) = mnist.load_data(delete=False)

# One-hot encode the array
y_train = np.eye(10)[y_train]

# Create a sequential model.
model = Sequential(
    [
        Flatten(),
        Dense(128, activation="relu"),
        Dropout(0.2),
        Dense(10, activation="softmax")
    ]
)

# Compile the model. Use mean squared error as our loss function.
model.compile(metrics=["accuracy"], loss="mse")

# Train the model.
model.fit(x_train, y_train, batch_size=250, epochs=1000, learning_rate=0.1)
```
